src/sentry/static/sentry/app/components/events/realUserMonitoring.tsx

```diff
@@ -11,13 +11,17 @@ import space from 'app/styles/space';
 import Tooltip from 'app/components/tooltip';
 import {IconFire} from 'app/icons';
 import {WEB_VITAL_DETAILS} from 'app/views/performance/realUserMonitoring/constants';
+import {Vital} from 'app/views/performance/realUserMonitoring/types';
 
 // translate known short form names into their long forms
 const LONG_MEASUREMENT_NAMES = {
   fid: 'First Input Delay',
   fp: 'First Paint',
   fcp: 'First Contentful Paint',
   lcp: 'Largest Contentful Paint',
+  cls: 'Cumulative Layout Shift',
+  ttfb: 'Time to First Byte',
+  'ttfb.requesttime': 'Request Time',
```

Comment:
These long names are already defined in `performance/realUserMonitoring/constants.tsx`, can we import them from there?

Reply:
should be fixed in https://github.com/getsentry/sentry/pull/21316/commits/311efda0cf35c60a6627aa0917154657c6645340